queries/tfvars/highlights.scm

```diff
@@ -0,0 +1,31 @@
+(comment) @comment
+
+(identifier) @constant
```

Comment:
Are they always constants?

Reply:
Yep, it's kinda `constants-file`

Reply:
https://github.com/opa-oz/tree-sitter-tfvars/blob/main/test/corpus/base.txt#L305

Starting here I grepped a bunch of real .tfvars from Github repos. No editing/function calls